src/app/page.tsx

```diff
@@ -127,110 +144,116 @@ export default function Home() {
         />
         <Link 
           href="/gallery"
-          className="btn-primary"
+          className="btn-primary mt-10"
         >
           Explore
         </Link>
       </div>
       {/* 2024 arts olympiad introduction */}
       {/* - logo and text - */}
-      <div className="grid grid-cols-1 gap-5 md:grid-cols-12 bg-secondary-blue col-span-1 md:col-span-10">
+      <div className="grid grid-cols-1 md:grid-cols-12 bg-secondary-blue col-span-1 md:col-span-10">
         <Image
           src="/home/torch-logo.svg"
           width={151}
           height={139}
           className="md:mt-16 md:ml-20 md:mb-10 md:col-span-2 col-span-1 justify-self-left mt-5 ml-5 mb-5"
           alt="2024 olympics torch ablaze"
         />
-        <div className="text-left text-xl ml-5 md:mt-36 md:mb-10 break-words md:col-span-4">
+        <div className="text-left text-xl ml-5 md:mt-36 md:mb-10 break-words md:col-span-5 md:ml-20">
```

Comment:
The spacing between this image and the text still looks a bit off (for example at a breakpoint of `800px`).